lib/protocol/bep_extensions.go

```diff
@@ -252,11 +269,26 @@ func (f FileInfo) isEquivalent(other FileInfo, comp FileInfoComparison) bool {
 			}
 		}
 		if f.Platform.Windows != nil && other.Platform.Windows != nil {
-			if *f.Platform.Windows != *other.Platform.Windows {
+			if f.Platform.Windows.OwnerName != other.Platform.Windows.OwnerName ||
+				f.Platform.Windows.OwnerIsGroup != other.Platform.Windows.OwnerIsGroup {
 				return false
 			}
 		}
 	}
+	if !comp.IgnoreXattrs && f.Platform != other.Platform {
+		if f.Platform.Linux != nil && other.Platform.Linux != nil && !xattrsEqual(f.Platform.Linux, other.Platform.Linux) {
```

Comment:
`xattrsEqual` could handle the nils for a bit less verbouse comparisons in this block.

Reply:
(done)